tests/ember-electron/main.js

```diff
@@ -47,15 +47,3 @@ app.on('ready', function onReady() {
     mainWindow = null;
   });
 });
-
-let cwd = __dirname || resolve(dirname());
```

Comment:
Uh wait, why did you remove this? IMO it's an important part of our "integration" testing that verifies that we are assembling the broccoli pipeline correctly.